tests/integration/test_repair_tools.py

```diff
@@ -285,6 +285,32 @@ def test_fix_interference(modeler: Modeler):
     assert result.success is True
 
```

Comment:
I am missing some tests using angles and distances for the new APIs added.. @RyanJWard @syscordan -- could you guys maybe take care of that as part of this PR (I think @jacobrkerstetter is out for the rest of the week)?